specification/iotcentral/resource-manager/Microsoft.IoTCentral/preview/2021-11-01-preview/examples/PrivateLinks_Get.json

```diff
@@ -0,0 +1,33 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "resRg",
+    "resourceName": "myIoTCentralApp",
+    "groupId": "iotApp",
+    "api-version": "2021-11-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/resRg/providers/Microsoft.IoTCentral/IoTApps/myIoTCentralApp/privateLinkResources/iotApp",
```

Comment:
The example is IoTApps. Could you check and make sure the example path consistent with swagger path, even the path is case insentive.

Reply:
I've made them all consistent as "iotApps" in the 2021-11-01-preview version. Thank you for pointing out the inconsistency.